apidocs/v0.4/index.html

```diff
@@ -4,6 +4,8 @@
 <meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
 <meta http-equiv="X-UA-Compatible" content="IE=9"/>
 <meta name="generator" content="Doxygen 1.8.11"/>
+<meta name="description" content="Website for Open Enclave SDK" />
+<meta name="Keywords" content="Microsoft,Open Enclave SDK"/>
```

Comment:
Can you please remove Microsoft from the keywords?
You are adding this to apidocs/v0.4, there are other versions of API docs for later releases. Is there a different place the keywords and description could be put?  Would putting it here make sense? https://github.com/openenclave/openenclave-site/blob/master/_layouts/default.html